website/pages/en/community.js

```diff
@@ -103,7 +103,7 @@ function ImplementationsShowcase() {
 function Implementations(props) {
 	const { config: siteConfig } = props;
 	const { repoUrl } = siteConfig;
-	const editUrl = `https://www.finos.org/get-involved-fdc3`;
+	const editUrl = `https://github.com/finos/FDC3/edit/main/website/`;
```

Comment:
I get a 404 for this URL as well... 

perhaps we should go back to the original URL: https://github.com/finos/FDC3/edit/master/website/data/community.json

or add instructions to the FDC3 readme and link to that?